@remirror/core-extensions/src/marks/link-extension.ts

```diff
@@ -99,31 +99,26 @@ export class LinkExtension extends MarkExtension<LinkExtensionOptions> {
       /**
        * A command to update the selected link
        */
-      updateLink: (attrs?: Attrs) => updateMark({ type, attrs }),
+      updateLink: (attrs?: Attrs): ProsemirrorCommandFunction => {
+        return (state, dispatch, view) => {
+          const { selection } = state;
+          if (selectionEmpty(selection) || (!isTextSelection(selection) && !isMarkActive({ state, type }))) {
```

Comment:
You might notice that before these changes isEnabled used
```
selectionEmpty(selection) || !isTextSelection(selection)
```
but here I have
```
selectionEmpty(selection) || (!isTextSelection(selection) && !isMarkActive({ state, type }))
```

Turns out that the former would incorrectly result in isEnabled=false for this test case